packages/frontend/src/app/invite/components/InviteForm.tsx

```diff
@@ -37,13 +38,15 @@ export function InviteForm({ authToken }: { authToken?: string }) {
   }
 
   return !isSubmitSuccessful ? (
-    <div>
-      <p className="text-2xl font-bold leading-normal pb-2">Invite a company</p>
-      <p className="text-sm leading-normal">
-        Enter email address of the company&apos;s representative,
-        <br />
-        that you want to invite.
-      </p>
+    <div className="flex flex-col gap-8">
+      <div className="flex flex-col gap-2">
+        <Text2xlBold>Invite a company</Text2xlBold>
+        <TextSm>
+          Enter email address of the company&apos;s representative,
+          <br />
```

Comment:
I think we shouldn't use `<br />` in such cases. The text should be naturally break by the size of the parent.